clients/src/main/resources/common/message/BeginQuorumEpochRequest.json

```diff
@@ -18,12 +18,21 @@
   "type": "request",
   "name": "BeginQuorumEpochRequest",
   "validVersions": "0",
-  "flexibleVersions": "0+",
   "fields": [
-    {"name": "ClusterId", "type": "string", "versions": "0+"},
-    {"name": "LeaderId", "type": "int32", "versions": "0+",
-      "about": "The ID of the newly elected leader"},
-    {"name": "LeaderEpoch", "type": "int32", "versions": "0+",
-      "about": "The epoch of the newly elected leader"}
+    { "name": "ClusterId", "type": "string", "versions": "0+"},
+    { "name": "Topics", "type": "[]BeginQuorumTopicRequest",
+      "versions": "0+", "fields": [
+      { "name": "TopicName", "type": "string", "versions": "0+", "entityType": "topicName",
+        "about": "The topic name." },
+      { "name": "Partitions", "type": "[]BeginQuorumPartitionRequest",
```

Comment:
I actually like current class names more, because if we do a direct import, we may still know this is `BeginQuorumPartitionRequest` not `EndQuorumPartitionRequest`, compared with confusing `PartitionData`

Reply:
The reason it stood out a little to me is that these names are missing `Epoch` which makes the names read a little bit odd. If you add `Epoch` back in, then you get `BeginQuorumEpochPartitionRequest` which is about the same length as `BeginQuorumEpochRequest.PartitionData`.